src/main/kotlin/com/exactpro/th2/rptdataprovider/messageProfile/MessageProfile.kt

```diff
@@ -0,0 +1,100 @@
+package com.exactpro.th2.rptdataprovider.messageProfile
+import com.exactpro.cradle.messages.StoredMessageId
+import com.exactpro.th2.rptdataprovider.entities.responses.Message
+import com.exactpro.th2.rptdataprovider.entities.responses.MessageWrapper
+import org.apache.log4j.*;
+import java.util.concurrent.atomic.AtomicInteger
+
+class MessageProfile(var messageId: StoredMessageId, var messageWrapper: MessageWrapper) {
+    private val logger = Logger.getLogger("myappender")
+    // время с начала оработки - не обнулять
```

Comment:
You shouldn't write comments in Russian. Also, do not duplicate information with a comment, which is clear from the name of the variable or the code of the method.